### ISSUE TITLE
Rewriter doesn't deal with division

### ISSUE THREAD

**Issue by wence-** [coneoproject/COFFEE#9]
The expression expanded doesn't know how to handle Div nodes.  Given that they essentially behave the same way as multiplication, can we just do:

```patch
diff --git a/coffee/rewriter.py b/coffee/rewriter.py
index 4c60b9e..c2d7468 100644
--- a/coffee/rewriter.py
+++ b/coffee/rewriter.py
@@ -563,7 +563,7 @@ class ExpressionExpander(object):
                 return ([node], self.ITVAR)
         elif isinstance(node, Par):
             return self.expand(node.children[0], node, itvars_occs, itvar_exp)
-        elif isinstance(node, Prod):
+        elif isinstance(node, (Prod, Div)):
             l_node, l_type = self.expand(node.children[0], node, itvars_occs, itvar_exp)
             r_node, r_type = self.expand(node.children[1], node, itvars_occs, itvar_exp)
             if l_type == self.ITVAR and r_type == self.ITVAR:
```
?

**Comment by wence-** [comment]
Can this fix go in?

**Comment by FabioLuporini** [comment]
Yes, I'm planning to do this and the other issue raised by Christian today, or at most tomorrow

**Comment by FabioLuporini** [comment]
Fix at the moment incorporated in branch fix-issues-8-18-19

**Comment by FabioLuporini** [comment]
Closing since fixed by PR #21
